platforms/documentation/docs/src/docs/userguide/authoring-builds/basics/plugins.adoc

```diff
@@ -234,6 +250,31 @@ plugins {
 }
 ----
 
+For example:
+
+====
+[.multi-language-sample]
+=====
+.build.gradle.kts
+[source, kotlin]
+----
+plugins {
+    id("com.gradleup.shadow") version "8.3.4"
```

Comment:
Would it make sense to also change this to a regular member function call rather than the inxix one? I mean `id("com.gradleup.shadow").version("8.3.4")`.

We have not made the decision to deprecate the infix notation here in Kotlin DSL, but the regular member call could potentially be easier to migrate to DCL in the future.

@eskatos WDYT?

Reply:
Let's address this on its own, outside this PR